tests/array/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -649,4 +662,23 @@ $arr = [
 $collapse = ["not", "associative"];
 $collapse = ["not", "associative"];
 
+$array = [
+    'key-one' => true ? 1 : 2,
+    'key-two' => 'VeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString'
+        ? 1
+        : 2,
+    'key-two' => true
+        ? 'VeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString'
+        : 2,
+    'key-two' => true
+        ? 1
+        : 'VeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString',
+    'key-two' => 'VeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString' .
+    'VeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString'
```

Comment:
Same indent with key is not related to this PR, we should improve printing `test` expression of `retif`, let's do it other PR